src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -398,6 +398,52 @@ class %s extends {$options['class']}
         return $code;
     }
 
+    private function isPreloadable(string $class): ?bool
+    {
+        static $cache = [];
```

Comment:
no static state please, a property would do fine isn't it?

Reply:
fixed